recipes/igwn-cmake/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "igwn-cmake" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://software.igwn.org/sources/source/{{ name }}-{{ version }}.tar.gz
+  sha256: c86ac84e53cbb141aaa58087de5e442bdb122db5af4c119dc7c89129f261da21
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - pkg-config
```

Comment:
I think you might need make here on unix, because it is used implicitly

Reply:
Done in 952e12b, thanks @scopatz.